package.json

```diff
@@ -1,11 +1,15 @@
 {
   "name": "eslint-config-canopy",
-  "version": "2.3.0",
+  "version": "3.0.0",
   "description": "A standard eslint configuration for Canopy frontend developers",
   "main": "lint-config.js",
   "repository": "git@canopy.githost.io:front-end/canopy-eslint.git",
   "license": "Apache-2.0",
   "dependencies": {
-    "eslint-plugin-react-hooks": "^1.0.0"
+    "@typescript-eslint/eslint-plugin": "^6.10.0",
+    "@typescript-eslint/parser": "^6.10.0",
+    "@babel/preset-typescript": "^7.23.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "typescript": "^5.2.2"
```

Comment:
Should "typescript" be under "peerDependencies"? If so I wasn't able to get this to work with `yarn link` unless it was in "dependencies", but it might be worth trying to test that out.

Reply:
best way to do this is to add it to both `devDeps` and `peerDeps` but only force a major version level for the peerDep

maybe `"typescript": "5"` ?

Reply:
kk we can try that out, but I also wasn't able to get this working when putting everything in devDeps. I figured that if the lib itself is included in devDeps that these dependencies would also be treated as devDeps but 🤷